modules/patterns/slave/slave.ino

```diff
@@ -46,69 +47,81 @@ typedef struct
 } t_pattern;
 
 static t_pattern const patterns[MAX_DIFFICULTY] = {
-  {{ 1, 0, 0 }, 2},
-  {{ 1, 1, 0 }, 1},
-  {{ 1, 1, 1 }, 2}
+  {{ 0, 1, 2 }, PIN_BTN_LEFT},
+  {{ 2, 0, 1 }, PIN_BTN_MIDDLE},
+  {{ 2, 2, 0 }, PIN_BTN_RIGHT}
 };
 
-static char const * const filenames[] = { "cat.bmp", "logo.bmp", "sample.bmp", "jeremy.bmp", "hexagon.bmp", "vim.bmp", "w2d.bmp", "resitor.bmp", "epitech.bmp" };
-static PImage images[] = { {}, {}, {}, {}, {}, {}, {}, {}, {} };
+#define NB_IMGS (3)
 
-#define sd_cs 4
-#define lcd_cs 10
-#define dc 9
-#define rst 8
+static char const * const filenames[] = { "cat.bmp", "logo.bmp", "vim.bmp", "sample.bmp", "hexagon.bmp", "vim.bmp", "w2d.bmp", "resitor.bmp", "jeremy.bmp", "epitech.bmp" };
+static PImage images[NB_IMGS] = { {}, {}, {} };
+
+#define sd_cs  (4)
+#define lcd_cs (10)
+#define dc     (9)
+#define rst    (8)
 
 // screen
 TFT TFTscreen = TFT(lcd_cs, dc, rst);
 
 void loadImages(void)
 {
```

Comment:
Might be good to use a local variable to store the amount of images as its useful 2 times.
With something like `size_t images_amount = sizeof(images) / sizeof(images[0])`, one must only play with the images array to updates images.

Clarity won't be touched but it would be a plus for maintainability.